resources/js/features/forums/components/ShortcodeRenderer/ShortcodeQuote/ShortcodeQuote.tsx

```diff
@@ -0,0 +1,26 @@
+import type { FC, ReactNode } from 'react';
+
+interface ShortcodeQuoteProps {
+  children: ReactNode;
+}
+
+export const ShortcodeQuote: FC<ShortcodeQuoteProps> = ({ children }) => {
+  // Remove leading <br>s and empty strings until we find content.
+  var isLeadingWhitespace = true;
```

Comment:
nit: Recommend using `let`.
```suggestion
  let isLeadingWhitespace = true;
```
This is supposed to be caught by a linting rule ... I will investigate why it isn't later today.

Reply:
Sorry, I didn't provide context. `var` has generally fallen out of favor in modern JS/TS because it doesn't respect block-scoping rules.